float64 x
float64 y
---
bool accepted
string message
